sources/packages/web/src/types/contracts/Assessment.ts

```diff
@@ -19,3 +23,34 @@ export enum AssessmentStatus {
    */
   declined = "Declined",
 }
+/**
+ * Assessment History Status
+ */
+export enum AssessmentHistoryStatus {
+  // When assessmentWorkflowId is null,
+  // then status is Submitted.
+  Submitted = "Submitted",
+
+  // When assessmentWorkflowId is not null
+  // and assessmentData is null, then status is
+  // InProgress.
+  InProgress = "In Progress",
+
+  // when assessmentWorkflowId is not null
```

Comment:
Noce work with the comments, please ensure that the sentences start with an upper case letter.